raycicmd/converter.go

```diff
@@ -118,6 +118,21 @@ func (c *converter) convertPipelineStep(step map[string]any) (
 		return s.buildkiteStep(), nil
 	}
 
+	// filter steps by tags
+	if step_tags, ok := step["tags"]; ok && filters != nil {
+		skip := true
+		for _, tag := range step_tags.([]interface{}) {
+			for _, filter := range filters {
+				if tag == filter {
+					skip = false
+				}
+			}
+		}
+		if skip {
+			return nil, nil // skip steps that are not tagged by filters
+		}
+	}
+
```

Comment:
rather than returning `nil` here, could you put the filtering in the caller of this function? so that tags can also work with wanda steps, wait steps or any other steps? or even entire rayci groups?